src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/TelemetryEventTypeResolver.kt

```diff
@@ -37,12 +33,6 @@ class TelemetryEventTypeResolver {
       updateTypes.add(GOAL_UPDATED)
     }
 
-    // check if the goal's status was updated
```

Comment:
As mentioned above, the goal status is no longer changed via a goal update operation ( we no longer support it at the AP) hence all this code can be removed,